### PR TITLE
Allow gyrotron controller to be buffered

### PR DESCRIPTION
**Pull request by DamianX**
Fixes .1 of #16190 

:cl:
 * rscadd: The gyrotron control computer can be buffered in multitools.

### CODE REVIEW

File: code/WorkInProgress/Cael_Aislinn/Rust/gyrotron.dm [hunk 1]
```diff
@@ -72,6 +72,16 @@
 		</ul>
 	"}
 
+/obj/machinery/rust/gyrotron/canLink(var/obj/machinery/computer/rust_gyrotron_controller/object, var/link/context)
+	. = (istype(object) && get_dist(src, object) < RUST_GYROTRON_RANGE)
```
Comment (PJB3005) [inline code/WorkInProgress/Cael_Aislinn/Rust/gyrotron.dm]: Use return, it's much more readable.

File: code/WorkInProgress/Cael_Aislinn/Rust/gyrotron_controller.dm [hunk 2]
```diff
@@ -1,13 +1,25 @@
-#define RUST_GYROTRON_RANGE 25
-
 /obj/machinery/computer/rust_gyrotron_controller
 	name = "Gyrotron Remote Controller"
 	icon_state = "engine"
 	circuit = /obj/item/weapon/circuitboard/rust_gyrotron_control
 	light_color = LIGHT_COLOR_BLUE
 
+	var/id_tag // Required for multitool buffering
```
Comment (PJB3005) [inline code/WorkInProgress/Cael_Aislinn/Rust/gyrotron_controller.dm:7]: Uh, is it actually? If it is then you should fix that instead.

File: code/WorkInProgress/Cael_Aislinn/Rust/~undefs.dm [hunk 3]
```diff
@@ -0,0 +1 @@
+#undef RUST_GYROTRON_RANGE
```
Comment (PJB3005) [inline code/WorkInProgress/Cael_Aislinn/Rust/~undefs.dm:1]: Don't bother honestly.

**Review by PJB3005 — CHANGES_REQUESTED** [review]

File: code/WorkInProgress/Cael_Aislinn/Rust/gyrotron_controller.dm (hunk 2 above)
Comment (DamianX) [reply code/WorkInProgress/Cael_Aislinn/Rust/gyrotron_controller.dm:7]: Indeed, I don't really want to do it right now, though.

File: code/WorkInProgress/Cael_Aislinn/Rust/gyrotron.dm (hunk 1 above)
Comment (N3X15) [reply code/WorkInProgress/Cael_Aislinn/Rust/gyrotron.dm]: Once this is fixed I'll merge.  Tiny thing but it makes a difference.

**Pull request merged**